extensions/wakatime/package.json

```diff
@@ -31,6 +31,22 @@
       "title": "Show Private Leaderboards",
       "description": "Show your private leaderboards",
       "mode": "view"
+    },
+    {
+      "name": "today-summary",
+      "subtitle": "WakaTime",
+      "title": "Show Today Summary",
+      "description": "Show Today's Summary in the Menu Bar",
+      "mode": "menu-bar",
+      "interval": "1m",
+      "preferences": [{
+        "name": "showAsksSentence",
+        "title": "Show Asks Sentence",
+        "description": "Show asks sentence for you when your coding time is zero",
```

Comment:
I don't think the message should be configurable but I'm not against it.

But there should be a `checkbox` preference on whether to show the menu bar item at all.